### PR TITLE
Fix inconsistencies for HRTIM_TIMF

### PR DESCRIPTION
**Pull request by usbalbin**
Aims to solve #847

Not really familiar with svdtools or even yaml for that matter.

Tested against modified stm32g4xx-hal for stm32g474 and atleast it seems to compile.

~~TODO: Should probably look through HRTIM_ADCER for TIMF-related errors...~~ - done

### CODE REVIEW

**Comment by usbalbin** [comment]
Hopefully solved in #847

File: devices/common_patches/hrtimv2.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  _modify:
+    RSTE1R:
+      name: RSTF1R
+HRTIM_Common:
+  OENR:
+    _modify:
+      TF1ODS:
+        name: TF1OEN
+        description: Bit 10 - Timer F Output 1 Enable
+        _read:
```
Comment (burrbull) [inline devices/common_patches/hrtimv2.yaml:11]: We keep `enumeratedValues` in `peripherals`.
Please split this file. Leave `_modify` part here and move enums in `peripherals`.

Comment (usbalbin) [reply devices/common_patches/hrtimv2.yaml:11]: Okay, thanks will (try to) do! :)

Comment (usbalbin) [reply devices/common_patches/hrtimv2.yaml]: I hope this better :)

**Comment by usbalbin** [comment]
Is there anything else I should fix? :)

**Comment by usbalbin** [comment]
I have now fixed my last TODO. I have also been using this for a while and it seems to be working as far as I can tell.

Let me know if there is anything else :)

**Comment by usbalbin** [comment]
@burrbull or anyone else, do you want me to rebase on top of latest stm32-rs/master or do you prefer to avoid force pushing until later?

Anything else to change/do differently?

File: devices/common_patches/hrtimv2.yaml (hunk 1 above)
Comment (usbalbin) [reply devices/common_patches/hrtimv2.yaml]: Anything at all I can do to help this move forward? :)

**Comment by adamgreig** [comment]
Looks good, thank you and sorry for the wait!

**Comment by usbalbin** [comment]
> Looks good, thank you and sorry for the wait!

Sorry, first time contributor here :)

Do I need to do something here like fix the changelog conflict, or what does the `Auto` in `Auto-merge` mean? :)

**Comment by usbalbin** [comment]
> Looks good, thank you and sorry for the wait!

@adamgreig am I supposed to fix the merge conflict or is there anything else I am expected to do?

**Comment by adamgreig** [comment]
Oh no, sorry! I enabled auto-merge and fixed the merge conflict but didn't remember to click 'approve', so I expected this to just get merged. Nothing for you to do, I'll fix up the new conflict and merge it.

**Review by adamgreig — APPROVED** [review]

**Comment by usbalbin** [comment]
Thank you! :)

**Pull request merged**